edenia/eden-spend-explorer
# fix(webapp): general and delegate views

### Update General and Delegate Views

### What does this PR do?

- Resolve #190 
- Resolve #192 

### Steps to test

1. Run project
2. Must see this:
![imagen](https://user-images.githubusercontent.com/49013556/198393486-5a677f2f-d91b-43c4-9b21-b63b57c314be.png)


#### CheckList
